### PR TITLE
Ignore organization name when fuzzy-finding in CD

### PR DESCRIPTION
**Pull request by selesse**
Given a repository structure of `Shopify/shopify` and `Shopify/web`, and an input of "shopify", the old implementation of `dev cd shopify` would always pick web. Why? The strings that were actually passed to fzy were `Shopify/web` and `Shopify/shopify`. Because `Shopify/web` is shorter, the matching score was slightly higher. This change modifies CD to only look at the individual repository names for `cd`'ing, rather than the full string. That way, `dev cd shopify` will prioritize repositories named `shopify` over organizations that are named `Shopify`.

In short, `dev cd shopify` will actually take us to the Shopify/shopify repository now when web is also cloned.

---

Illustration of the problem:

```
$ for x in Shopify/shopify Shopify/web Shopify/storefront-renderer; do echo $x; done | fzy --show-matches=shopify --show-scores
6.880000	Shopify/web
6.860000	Shopify/shopify
6.800000	Shopify/storefront-renderer
```

### CODE REVIEW

**Comment by burke** [comment]
I see your point but I don't think this is quite right either... I'll often find myself doing things like `dev cd b/dev` to get to this repo. It's less common in the spin workflow that we're using this for though... really I think we should probably be bringing in some more sophisticated matching than this. I'm not really sure... @lugray do you have thoughts on this?

**Comment by lugray** [comment]
I agree that we want to keep matching on the whole thing. To solve the issue, I'd want to bring in full-dev's notion of frecency.

**Comment by selesse** [comment]
I think there's a middle-ground solution here where we only include the organization if we use a `/` in the path. I'd be curious to know how often people use that form, I'd expect most use cases (especially at Shopify) are just fuzzy finding on the repo name.

I'll close this out though :)
